### PR TITLE
Development

### PR DESCRIPTION
**Pull request by phuongnguyen00**
Add the deleting all comments feature to step 6 of week 3.

### CODE REVIEW

**Comment by ApsarasMax** [comment]
Is this ready for code review?

**Comment by phuongnguyen00** [comment]
> Is this ready for code review?
Yes (for last week's code review). I'm only starting on this week.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -49,7 +53,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     List<Comment> comments = new ArrayList<>();
-    for (Entity entity : results.asIterable()) {
+    for (Entity entity : results.asIterable(FetchOptions.Builder.withLimit(MAX_COMMENTS_NUM))) {
```
Comment (ApsarasMax) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I was wondering whether there could be ways to restrict the max_num in query itself but looking at the API, it seems not.

Comment (phuongnguyen00) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I actually changed that later on when I add a query string to the request sent to the servlet.

**Pull request merged**